src/util/nativeStorageHelper.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable fp/no-class */
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+/**
+ *
+ */
+function handleError(func: string, param?: string): Promise<string> {
+  let message
+  if (!param) {
+    message = func
+  } else {
+    message = `${func}() requires at least ${param} as its first parameter.`
+  }
+  console.warn(message) // eslint-disable-line no-console
+  return Promise.reject(message)
+}
+
+/**
+ *
+ */
+class SyncStorage {
+  data: Map<string, string> = new Map()
+
+  loading = true
+
+  async init(): Promise<void> {
+    await AsyncStorage.getAllKeys().then((keys: string[]) =>
+      AsyncStorage.multiGet(keys).then((data: any): any[] => {
+        data.forEach(this.saveItem.bind(this))
+
+        return [...this.data]
```

Comment:
I don't think the return value goes anywhere. Did you mean to return it from `init`, or is not used?